include/spm/fadedrv.h

```diff
@@ -1,10 +1,26 @@
 #pragma once
+#include <wii/types.h>
 
 namespace spm::fadedrv {
 
+struct FadeWork {
+    u8 _00[0x28-0x00];
+    wii::Vec3 fadeCenterPosition;
```

Comment:
The center position is part of an array of substructs, just pushed a basic skeleton of them to the PAL Ghidra

Reply:
I'll just declare it unknown next commit then.